package.json

```diff
@@ -73,6 +73,7 @@
     "tmp": "0.0.30",
     "watchpack": "1.3.0",
     "update-notifier": "2.0.0",
+    "watchpack": "1.2.1",
```

Comment:
this line should be removed (watchpack is already included in the dependencies at line 74)